aten/src/ATen/cpu/vec/vec256/vec256_float.h

```diff
@@ -97,6 +97,29 @@ template <> class Vectorized<float> {
       std::memcpy(ptr, tmp_values, count * sizeof(float));
     }
   }
+  void solution2_satuate(uint8_t* dst_data) {
```

Comment:
typo? should be `saturate`?

Reply:
Oh, yean. Thanks. This PR is still work in progress. I will refine it sooner.

Reply:
Thanks, the typo is fixed.